.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
+omit =
+    # Don't complain if non-runnable code isn't run:
+    */__main__.py
+
+[report]
+exclude_lines =
+    # Have to re-enable the standard pragma
+    \#\s*pragma: no cover
```

Comment:
there are some parts of the CLI that are not intended to be covered by (unit)tests